app/src/main/res/values-fr/sura_names.xml

```diff
@@ -1,5 +1,122 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <string-array name="sura_names">
+    <item>AL-FÃTIHAH</item>
+    <item>AL-BAQARAH</item>
+    <item>ÃL-`IMRÃN</item>
+    <item>AN-NISÃ\'</item>
+    <item>AL-MÃ-\'IDAH</item>
+    <item>AL-AN`ÃM</item>
+    <item>AL-A`RÃF</item>
+    <item>AL-ANFÃL</item>
+    <item>AT-TAWBAH</item>
+    <item>YOUNUS</item>
+    <item>HOUD</item>
+    <item>YOUSUF</item>
+    <item>AR-RA`D</item>
+    <item>IBRÃHÎM</item>
+    <item>AL-HIJR</item>
+    <item>AN-NAHL</item>
+    <item>AL-ISRÃ\'</item>
+    <item>AL-KAHF</item>
+    <item>MARЇAM</item>
+    <item>TÃ-HÃ</item>
+    <item>AL-ANBIYÃ\'</item>
+    <item>AL-HAJJ</item>
+    <item>AL-MU\'MINOUN</item>
+    <item>AN-NOUR</item>
+    <item>AL-FURQÃN</item>
+    <item>ACH-CHU`ARÃ\'</item>
```

Comment:
@nacer80 also, do all people who speak French use the CH instead of sh?

Reply:
Yes, in Franch language they use "ch" it pronanced as "sh" in English